apis/v1alpha1/generator.yaml

```diff
@@ -0,0 +1,43 @@
+resources:
+  ParameterGroup:
+    exceptions:
+      errors:
+        404:
+          code: ParameterGroupNotFoundFault
+      terminal_codes:
+        - ParameterGroupQuotaExceededFault
+        - ParameterGroupAlreadyExistsFault
+        - ServiceLinkedRoleNotFoundFault
+        - InvalidParameterGroupStateFault
+        - InvalidParameterValueException
```

Comment:
Is this terminal? Is it possible to change the spec to undo this exception?

Reply:
These are terminal errors, spec can undo all of them  except `ServiceLinkedRoleNotFoundFault` will remove it from the list.